src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-31">
+      <div class="calendar__day calendar__day--1 start-sun"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```
you don't need those classes